src/seq2seq.ipynb

```diff
@@ -0,0 +1,686 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 56,
```

Comment:
Error appears on github with  the following: Invalid Notebook: 'execution_count' is a required property, i'm unsure why